### ISSUE TITLE
Assassin's Creed Origins broken graphics (with apitrace)

### ISSUE THREAD

**Issue by thirdeyefunction** [doitsujin/dxvk#540]
As title, graphics are broken with DXVK under native Windows, as was discussed in #535. Game doesn't work with Wine at all, likely due to the game's convoluted DRM scheme.

### Software information
Assassin's Creed Origins, default settings

### System information
- GPU: Nvidia 980ti
- Driver: 398.82 (Windows)
- Wine version: N/A 
- DXVK version: Git master

### Apitrace file(s)
- [Windows apitrace](https://mega.nz/#!DXJ2BShY!bC0IkbL1lQkxgdncVO4QBGQogVvnxI0TxlHvPik_iQg)

### Log files
- d3d11.log:  [d3d11 log](https://pastebin.com/UWRneMpn)
- dxgi.log: [dxgi log](https://pastebin.com/ZbCJ763Q)

**Comment by doitsujin** [comment]
Thanks for recording a trace, but can you please upload it somewhere else (such as Mega, Google Drive, or Dropbox)? The file hoster you picked won't let me download more than half of it.

**Comment by thirdeyefunction** [comment]
My bad, I fixed the link.

**Comment by doitsujin** [comment]
Thanks. Sadly, replaying the trace on my system causes a GPU hang, which means that I won't be able to do anything at all until the RADV/LLVM issues get fixed.

**Comment by doitsujin** [comment]
Something in the logs bothers me:
```err:   DxvkPhysicalBufferView: Failed to create buffer view:
  Offset: 0
  Range:  461373440
  Format: VK_FORMAT_R32_UINT```

This is the same issue as in #537, #366. It looks like Nvidia reverted the fix for the driver bug which is causing these issues.

**Comment by thirdeyefunction** [comment]
Good call, the Windows vulkan beta (398.91) has less broken graphical output, so presumably has that fix.

**Comment by doitsujin** [comment]
Game should render fine with latest `master` if you use a driver that includes the buffer view fix (see #278).
